win32/devdoc/gballoc_ll_passthrough.md

```diff
@@ -76,3 +78,13 @@ MOCKABLE_FUNCTION(, void*, gballoc_ll_realloc, void*, ptr, size_t, size);
 **SRS_GBALLOC_LL_PASSTHROUGH_02_006: [** `gballoc_ll_realloc` shall call `realloc(nmemb, size)` and return what `realloc` returned. **]**
 
 
+### gballoc_ll_size
+```c
+MOCKABLE_FUNCTION(, size_t, gballoc_ll_size, void*, ptr);
+```
+
+`gballoc_ll_size` returns 0 because stdlib has no API that provides back to the user the size of the allocated block.
+
+**SRS_GBALLOC_LL_PASSTHROUGH_02_007: [** `gballoc_ll_size` shall return what `_msize` returns. **]**

```

Comment:
return 0 or return "what _msize returns"?

Reply:
Spec text is authoritative :)

line 86 shall change

---
In reply to: [459658810](https://github.com/Azure/azure-c-pal/pull/14#discussion_r459658810) [](ancestors = 459658810)